server/functions/app.mjs 

```diff
@@ -0,0 +1,10 @@
+import express from "express";
```

Comment:
Why is the extension of this file `mjs`?

Reply:
we will not use it so I have deleted the file